met-web/src/components/layout/Table.tsx

```diff
@@ -0,0 +1,436 @@
+import * as React from "react";
```

Comment:
Noticed several unused imports, unused methods, commented out blocks of code. Also it would be good if all our functions remained arrow functions for consistency. Component seems to need some cleanup from all the extra features in it that are not used as well.